src/client/ui/viewport/ViewportPanelContainer.js

```diff
@@ -29,94 +24,15 @@ class ViewportPanelContainer extends Component {
     this.props.editor.createViewport(this.canvasRef.current);
   }
 
-  onDropAsset = async item => {
```

Comment:
Removed support for dropping urls and files?

Reply:
Yeah, unfortunately it already wasn't really working. I plan to add proper support back in soon.